torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4609,7 +4609,7 @@ def _test_ddp_apply_optim_in_backward(
                 with torch.backends.cudnn.flags(
                     enabled=True, deterministic=True, benchmark=False
                 ):
-                    for i in range(100):
```

Comment:
why we need to change the test iteration?

Reply:
I initially added 100 to ensure determinism, but feel like 100 iterations increases TTS a bit too much. Can revert this back if needed.